test/openai/test_completion.py

```diff
@@ -444,7 +444,7 @@ def success_metrics(responses, solution, **args):
         vanilla_config = {
             "model": "gpt-3.5-turbo",
             "temperature": 1,
-            "max_tokens": 2048,
+            "max_tokens": 16,
```

Comment:
Why is this change needed? FYI, for gpt-3.5-turbo, the default max_tokens is inf.